x-pack/plugins/security_solution/cypress/screens/alerts_detection_rules.ts

```diff
@@ -75,7 +75,7 @@ export const RULES_TABLE = '[data-test-subj="rules-table"]';
 
 export const RULES_ROW = '.euiTableRow';
 
-export const RULES_MONITORING_TABLE = '[data-test-subj="allRulesTableTab-monitoring"]';
+export const RULES_MONITORING_TABLE = '[role="tablist"] > [data-test-subj="navigation-monitoring"]';
```

Comment:
What was the reason for this change?

FYI it's still used in `x-pack/test/security_solution_ftr/page_objects/detections/index.ts`:

```ts
  async navigateToRuleMonitoring(): Promise<void> {
    await this.common.clickAndValidate('allRulesTableTab-monitoring', 'monitoring-table');
  }
```

Reply:
I've changed the tab implementation to use another component which already implements all the necessary functionality to track path segment and activate a proper tab. It's used already at the rule details page.

The component has a different scheme for setting up `data-test-subj` so changing it would break more than fix so `RULES_MONITORING_TABLE` selector was updated as a side effect.

Reply:
I've looked at `x-pack/test/security_solution_ftr/page_objects/detections/index.ts` and I wasn't able to find any code using `navigateToRuleMonitoring()`. On top of that selector in `navigateToExceptionList()` look outdated as well

```
  async navigateToExceptionList(): Promise<void> {
    await this.common.clickAndValidate('allRulesTableTab-exceptions', 'exceptions-table');
  }
```

Maybe it makes sense to remove unused methods?

Reply:
@maximpn Got it, thank you! Yes, let's delete the unused methods: `navigateToRuleMonitoring` and `navigateToExceptionList`.